pkg/api/v1/types.go

```diff
@@ -653,9 +659,17 @@ type Container struct {
 	// Required.
 	Image string `json:"image" description:"Docker image name"`
 	// Optional: The docker image's entrypoint is used if this is not provided; cannot be updated.
-	Command []string `json:"command,omitempty" description:"entrypoint array; not executed within a shell; the docker image's entrypoint is used if this is not provided; cannot be updated"`
+	// Variable references $(VAR_NAME) are expanded using the container's environment.  If a variable
+	// cannot be resolved, the reference in the input string will be unchanged.  The $(VAR_NAME) syntax
+	// can be escaped with a double $$, ie: $$(VAR_NAME).  Escaped references will never be expanded,
+	// regardless of whether the variable exists or not.
+	Command []string `json:"command,omitempty" description:"entrypoint array; not executed within a shell; the docker image's entrypoint is used if this is not provided; cannot be updated; variable references are expanded according to the expansion spec"`
```

Comment:
Hrm, i thought I changed them to a and b. it's possible I missed them.
Thanks for review
On Sat, May 23, 2015 at 7:35 PM Alex Robinson <notifications@github.com>
wrote:

> In pkg/api/v1/types.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8149#discussion_r30949076>
> :
>
> > @@ -653,9 +659,17 @@ type Container struct {
> >  	// Required.
> >  	Image string `json:"image" description:"Docker image name"`
> >  	// Optional: The docker image's entrypoint is used if this is not provided; cannot be updated.
> > -	Command []string `json:"command,omitempty" description:"entrypoint array; not executed within a shell; the docker image's entrypoint is used if this is not provided; cannot be updated"`
> > +	// Variable references $(VAR_NAME) are expanded using the container's environment.  If a variable
> > +	// cannot be resolved, the reference in the input string will be unchanged.  The $(VAR_NAME) syntax
> > +	// can be escaped with a double $$, ie: $$(VAR_NAME).  Escaped references will never be expanded,
> > +	// regardless of whether the variable exists or not.
> > +	Command []string `json:"command,omitempty" description:"entrypoint array; not executed within a shell; the docker image's entrypoint is used if this is not provided; cannot be updated; variable references are expanded according to the expansion spec"`
>
> Ah, looks like you just missed the two in this file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8149/files#r30949076>
> .
>